src/main/java/org/assertj/guava/api/RangeAssert.java

```diff
@@ -256,16 +256,16 @@ protected RangeAssert(final Range<T> actual) {
    * assertThat(range).contains(10, 11, 12);</code></pre>
    *
    * @param values the values to look for in actual {@link com.google.common.collect.Range}.
-   * @return this {@link OptionalAssert} for assertions chaining.
+   * @return this {@link RangeAssert} for assertions chaining.
    * @throws AssertionError if the actual {@link com.google.common.collect.Range} is {@code null}.
    * @throws AssertionError if the actual {@link com.google.common.collect.Range} does not contain the given values.
    */
-  public RangeAssert<T> contains(@SuppressWarnings("unchecked") final T... values) {
+  public RangeAssert<T> contains(final T... values) {
```

Comment:
why remove the suppress warning? I still have a compilation warning on this line

Reply:
Sorry, my mistake.

At first I tried to update all methods with varargs with @SafeVarargs and final modifier. Then I realized, that class with these methods cannot be used in proxies. I've reverted back annotations and modifier, but I forget about @SuppressWarnings.